include/opcode/riscv.h

```diff
@@ -395,8 +395,11 @@ struct riscv_opcode
      INSN_MACRO, then this field is the macro identifier.  */
   insn_t mask;
   /* A function to determine if a word corresponds to this instruction.
-     Usually, this computes ((word & mask) == match).  */
-  int (*match_func) (const struct riscv_opcode *op, insn_t word);
+     Usually, this computes ((word & mask) == match).  If the constraints
+     checking is disable, then most of the function should check only the
+     basic encoding for the instruction.  */
+  int (*match_func) (const struct riscv_opcode *op, insn_t word,
+		     int contraints);
```

Comment:
typo, `contraints`->`constraints`, and `bfd_boolean` for constraints would be better.

Reply:
Fixed